src/main.ts

```diff
@@ -26,10 +26,17 @@ const getOutputWriter = async (
       if (output === 'google-sheets') {
         return createGoogleSheetsOutputWriter(initialUrl);
       }
+      const googleSheetsPrefix = 'google-sheets:';
```

Comment:
Potential change. (I don't feel strongly about this but figured I'd mention it. Feel free to ignore)

```suggestion
      const googleSheetsPrefix = 'google-sheets:';
      if (output === googleSheetsPrefix) {
        return createGoogleSheetsOutputWriter(initialUrl);
      }
```

Reply:
Then it wouldn't detect it without the colon, you'd have to do `-o google-sheets:` for it to infer the title.

We could do:

```ts
const googleSheetsPrefix = 'google-sheets'; // (without colon)
```

and then later do:

```ts
if (output.startsWith(googleSheetsPrefix + ':')) {
  const title = output.slice(googleSheetsPrefix.length + 1)
  ...
}
```
but that feels harder to read for me

Reply:
Oh, I hadn't noticed the colon! That's fair. Works for me :+1: